library/std/arithmetic_internal.qs

```diff
@@ -1,36 +1,127 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Microsoft.Quantum.Arithmetic {
+namespace Microsoft.Quantum.Unstable.Arithmetic {
```

Comment:
Based on the file name, was this the intended namespace?

Reply:
Yes. All these functions are unstable. And internal.